pkg/container-collection/k8s.go

```diff
@@ -45,6 +45,7 @@ type K8sClient struct {
 	nodeName      string
 	fieldSelector string
 	runtimeClient runtimeclient.ContainerRuntimeClient
+	SocketPath    string
```

Comment:
```suggestion
	RuntimeSocketPath    string
```

For a second I thought it was a socket path to talk to the API server.